odh-common/base/odhdocumentcrd.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: odhdocuments.documents.odh.io
```

Comment:
Should we specify that this is for the `dashboard` or is there some future plan to implement a generic `odhdocument` that provides some additional functionality beyond the dashboard.

Some variation of `ODHDocument.dashboard.opendatahub.io`

Reply:
looks good thanks